activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -92,6 +105,7 @@ def column_options(o)
             column_options[:auto_increment] = o.auto_increment
             column_options[:primary_key] = o.primary_key
             column_options[:collation] = o.collation
+            column_options[:comment] = o.comment if supports_comments?
```

Comment:
I think `if supports_comments?` is not needed.
For example, `:first`, `:after` and `:auto_increment` are only handled by mysql2 adapter (not handled by postgresql and sqlite3 adapters). `:collation` is only handled by mysql2, postgresql and sqlite3 adapters for now.